java/ql/lib/ext/java.lang.module.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["java.lang.module", "ModuleReader", True, "find", "(String)", "", "Argument[0]", "read-file", "ai-generated"] # The documentation implies that the location is actually read: https://docs.oracle.com/javase/9/docs/api/java/lang/module/ModuleReader.html#find-java.lang.String-
```

Comment:
The danger isn't in the I/O operation itself, but rather in whether the method returns the read contents in some way. Since this returns a URI, should we make it a taint step instead (similarly to the `URI` constructor)?